components/google_sheets/google_sheets.app.mjs

```diff
@@ -45,7 +49,6 @@ export default {
     sheetID: {
       type: "string",
       label: "Spreadsheet",
-      description: "The Google spreadsheet",
```

Comment:
I'm seeing ESLint errors for props without descriptions.
```
Component prop sheetID must have a description. See https://pipedream.com/docs/components/guidelines/#props eslint(pipedream/props-description)
```

To fix these errors, I'd suggest to either:
- Add a [configuration comment](https://eslint.org/docs/user-guide/configuring/rules#disabling-rules) to disable this rule
```js
    // eslint-disable-next-line pipedream/props-description
    sheetID: {
```
- Or set the description to an empty string?
```
      description: "",
```

Reply:
Humm, My VSCode is not showing this error for me, fixed